septentrion/db.py

```diff
@@ -55,7 +55,15 @@ def execute(
     args: Tuple = tuple(),
     commit: bool = False,
 ) -> Any:
-    query = " ".join(query.format(table=settings.TABLE).split())
+    query = " ".join(
+        query.format(
+            table=settings.TABLE,
+            version_column=settings.VERSION_COLUMN,
+            name_column=settings.NAME_COLUMN,
+            applied_column=settings.APPLIED_COLUMN,
+        ).split()
```

Comment:
Dumb question, why do we need to split & join the query? :)

Reply:
to remove line breaks.

in the past we had : https://github.com/peopledoc/septentrion/commit/76138bb6ed060f9b82f68ca2d3875c5cf03881d2#diff-dcf2e67bd860136fb966a5112a6ab275L17

Reply:
Why do we need to remove line break, then ?

Reply:
You wrote the original code, so you tell me :) 
Maybe it's not useful anymore, I didn't dig into that, as it's not the subject of that PR...

Reply:
Wow, it's hard to remember things from 2 years ago.

My guess would be a similar project I was inspired from was doing the same. I'm trying to remember if we were working on postgres tools at the time...

Today, my bet would be: if nothing justifies it, let's get rid of it. If we needed, we'll see it soon enough and re-add it with a comment !

Reply:
Clarifying! Let's get rid of it **in another PR, some other time** https://github.com/peopledoc/septentrion/issues/71